website/views/shared/profiles/stats.jade

```diff
@@ -23,13 +23,20 @@ table.table.table-striped
 
 // FIXME get translations working before can use this
 unless mobile
-  h4.stats-equipment(class=mobile?'item item-divider':'',ng-show='user.flags.itemsEnabled')=env.t('equipment')
+  h4(class=mobile?'item item-divider':'',ng-show='user.flags.itemsEnabled')=env.t('battleGear')
   table.table.table-striped(ng-show='user.flags.itemsEnabled')
     tr(ng-repeat='(k,v) in profile.items.gear.equipped', ng-init='piece=Content.gear.flat[v]', ng-show='piece')
       td
         strong {{piece.text()}}:&nbsp;
         span(ng-repeat='stat in ["str","con","per","int"]', ng-show='piece[stat]') {{piece[stat]}} {{stat.toUpperCase()}}&nbsp;
 
+unless mobile
+  h4(class=mobile?'item item-divider':'',ng-show='profile.preferences.costume && user._id != profile._id')=env.t('costume')
```

Comment:
Both of these checks for adding the `item item-divider` classes shouldn't be here. Since they're nested underneath unless mobile, mobile will never be true. It's also unnecessary to have both `unless mobile` lines. You can remove this one as it'll still be under the first one.